### ISSUE TITLE
Option that the received pictures dont get shrinked/cropped down?

### ISSUE THREAD

**Issue by TomTheDragon** [EionRobb/skype4pidgin#407]
I know that some maybe like not having a super big picture in their window if someone sends them one but its just not practicable for me, since the pictures also get cropped down to a 1:1 format.
So how about an option to receive the full image instead? (just like the telegram plugin does)

**Comment by camel1cz** [comment]
Yes, that also bothers me a lot... it makes the functionality often useless :(
Would it be possible to at least scale the image to "best fit" into the rectangular area of the current image?
And if the currently working "Save Image..." functionality on the right mouse click would save the original picture, it would be just perfect!

**Comment by EionRobb** [comment]
The telegram prpl does it slightly differently, that it'll download the image locally into the .purple folder.  We could do the same in the Skype plugin perhaps?  We could also provide a link to the online version of the image, but it'd need you to login to your Skype account to view it?

**Comment by xvitaly** [comment]
> The telegram prpl does it slightly differently, that it'll download the image locally into the .purple folder. We could do the same in the Skype plugin perhaps?

I don't think it is a good idea. I think plugin should download such files into system's temporary directory.

**Comment by TomTheDragon** [comment]
Hm.. this reminds me of the linux version of Skype, where you have to log in too.
I would say that it would be the best to have the option to do both, because it would be a problem to see any pictures if youre running the plugin on a medium who dont supports displaying pictures, like Spectrum2 for example.
